x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -636,6 +667,28 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
       }
     }
 
+    if (props.sourceIndicesWithGeoFields[anomaly.jobId]) {
+      items.push(
+        <EuiContextMenuItem
+          key="view_in_maps"
+          icon="gisApp"
+          onClick={async () => {
+            const mapsLink = await getAnomalySourceMapsLink(
+              anomaly,
+              props.sourceIndicesWithGeoFields
+            );
+            await application.navigateToApp(MAPS_APP_ID, { path: mapsLink?.path });
+          }}
+          data-test-subj="mlAnomaliesListRowActionViewSourceIndexInMapsButton"
+        >
+          <FormattedMessage
+            id="xpack.ml.anomaliesTable.linksMenu.viewSourceIndexInMapsLabel"
+            defaultMessage="View source index in Maps"
```

Comment:
Depending on whether you retain both menu items for jobs that use a geo field in the job config, this could be simplified to `View in Maps`.

Reply:
Decided not to retain both items and kept the original link label so it's clear they're looking at the source index and not the anomaly